packages/idds/idds.0.2/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "-p" name "@doc"] {with-doc}
+]
+maintainer: ["Steffen Smolka <smolka@cs.cornell.edu>"]
+authors: ["Steffen Smolka <smolka@cs.cornell.edu>"]
+bug-reports: "https://github.com/netkat-lang/idds/issues"
+homepage: "https://github.com/netkat-lang/idds"
+doc: "https://netkat-lang.github.io/idds/"
+license: "MIT"
+version: "0.2"
+dev-repo: "git+https://github.com/netkat-lang/idds.git"
+synopsis: "Identity-suppressed decision diagrams (IDDs)"
+description: "BDD and IDD implementation with hash-consing"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.10"}
```

Comment:
```suggestion
  "dune" {>= "1.10"}
```
Dune should not be tagged as a build dependency (see https://github.com/ocaml/opam-repository/pull/14266)